### ISSUE TITLE
Bundle missing error

### ISSUE THREAD

**Issue by themanro** [JohnCoates/CSSketch#35]
![screen shot 2016-03-17 at 10 25 32](https://cloud.githubusercontent.com/assets/2836170/13849002/a4fb2a32-ec2a-11e5-8936-800715d3ff07.png)

Getting this error when trying to run Layout Layers.

**Comment by motleydev** [comment]
Ok, after digging through this myself and having lots of issues - I realized that the SketchToolbox install doesn't work - you need to install from the "latest" distribution. There's a direct link in readme. Now it works completely fine. Here's my support request for getting toolbox support back up!! Sweet plugin. My pie in the sky wish would be a feature that generates a stylesheet based on the layer names in an existing folder and EVEN get those values and create the stylesheet for you - but.... That's a heck-a-ton of work. :)

**Comment by andrewmbacon** [comment]
Motleydev is right, just need to install it manually.

**Comment by JohnCoates** [comment]
Sorry about that! SketchToolbox install support will be re-established soon

**Comment by kunoxs** [comment]
I have the same error. I already deleted the plugin from the Sketch Toolbox and then installed CSSketch manually but the error message is still there. Already restarted Sketch and did the same process several times without success

**Comment by lkoeln** [comment]
for me it helped to re-install Sketch itself, too @kunoxs

**Comment by kunoxs** [comment]
@lkoeln Looked like a possible solution but it did not work :( danke sehr anyway

**Comment by JohnCoates** [comment]
If you're running CSSketch.sketchplugin after downloading the source or
.zip it won't work. Make sure you're running the installer from
https://github.com/JohnCoates/CSSketch/releases/latest

**Comment by kunoxs** [comment]
@JohnCoates Yes but I'm using OS X 10.10.5 and that installer is for 10.11 so is not working
